listings/ch00-introduction/struct_as_mapping_key/src/lib.cairo

```diff
@@ -0,0 +1,26 @@
+#[contract]
+mod StructAsMappingKey {
+    use starknet::get_caller_address;
+    use starknet::ContractAddress;
+    use poseidon::poseidon_hash_span;
```

Comment:
Is this the correct function that I should use? @enitrat

Reply:
yes